lib/QuipProcessor.js

```diff
@@ -295,8 +295,27 @@ class QuipProcessor {
                 dateStr = updatedDate === createdDate? createdDate : `${createdDate} (Updated: ${updatedDate})`;
                 dateStr = `, ${dateStr}`;
             }
+
+            let referencedSections = "";
+            if(message.annotation && message.annotation.id) {
+                if(message.annotation.highlight_section_ids) {
+                    referencedSections = message.annotation.highlight_section_ids.map((sectionId, index) => {
+                        // Not sure if there can be more than one highlight section,
```

Comment:
So far in my company's data I've only seen one such case, but for whatever reason it ended up pointing back to the same section reference. So it might make sense to wrap this in a `Set` to avoid duplicated links.